wolf/ml/referee_ocr/ocr_referee_class.h

```diff
@@ -0,0 +1,192 @@
+#ifndef OCR_REFEREE_CLASS_H
```

Comment:
all these files should be moved under `wolf::ml::ocr` namespace.
also make sure each name should be started with `w_` . (e.g. `w_ocr_referre_class.h`>
please don't use `class` or `name of namespace` for file naming; for example, the name `w_referre` is preferred rather than `w_ocr_referre_class`.